### PR TITLE
use SG index defined in ES config

### PR DESCRIPTION
**Pull request by jcantrill**
This PR
* modified seeding to get the SG index from the ES config

cc @stevekuznetsov

### CODE REVIEW

**Comment by jcantrill** [comment]
[test]

File: elasticsearch/run.sh [hunk 1]
```diff
@@ -151,9 +151,13 @@ exit 1
 
 seed_searchguard(){
     info Seeding the searchguard ACL index
+
+    config=/usr/share/elasticsearch/config/elasticsearch.yml
```
Comment (richm) [inline elasticsearch/run.sh]: I think this path is different in origin vs. enterprise?

Comment (jcantrill) [inline elasticsearch/run.sh]: probably... i feel like there is a bunch of settings that should probably be moved to some sort of ENV script

**Comment by richm** [comment]
[merge]

**Comment by jcantrill** [comment]
backport 1.5 in #519

**Pull request merged**